Automattic/jetpack
# Media Uploads: Set Max File Upload Size to PHP Const

Summary: This diff replaces references to `fileupload_maxk` in the media endpoints, and filters the options due to references to it that exist in core (`wp-admin/includes/ms.php`, `wp-includes/ms-functions.php` and `wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php`) to increase our max allowed upload size to the actual max value as defined in `WPCOM_MAX_UPLOAD_FILE_SIZE`.

Test Plan:
   1. sandbox a site with a VideoPress plan
   2. don't sandbox the transcode jobs, they will take too long and eat up your resources for this test
   3. upload a file larger than 1.5GB from the following locations (note, you may need to clear your `/tmp` in the sandbox if you run out of space):
      - `/wp-admin/upload.php`
      - `/wp-admin/media-new.php`
      - `wordpress.com/media/<site>`
      - from the `/video` block in the editor
   4. You should not get any errors about the file being too big
   5. run core tests to ensure they still pass `cd bin/tests/core && phpunit tests/wpcom-core-compat/rest-api/rest-attachments-controller.php`

Reviewers: #greenhouse_team, danroundhill

Reviewed By: #greenhouse_team, danroundhill

Subscribers: danroundhill

Tags: #touches_jetpack_files

Differential Revision: D69216-code

This commit syncs r234156-wpcom.




Fixes #

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Ensure jetpack tests all pass?
* This file doesn't appear to actually get loaded as it contains a bunch of WPCOM only code.
*
